gpMgmt/bin/lib/gp_bash_functions.sh

```diff
@@ -213,10 +213,11 @@ NORMALIZE_CODESET_IN_LOCALE () {
 	echo "$language_and_territory$([ ! -z $codeset ] && echo ".$codeset")$([ ! -z $modifier ] && echo "@$modifier")"
 }
 
-IN_ARRAY () {
+LOCALE_IS_AVAILABLE () {
 	local locale=$(NORMALIZE_CODESET_IN_LOCALE $1)
```

Comment:
Rename `locale` variable to smth more appropriate form so it's meaning doesn't conflict with `$1`